README.md

```diff
@@ -1,44 +1,42 @@
-# Redmadrobot Android KTX
- [![Build Status](https://img.shields.io/github/workflow/status/RedMadRobot/redmadrobot-android-ktx/CI/main?style=flat-square)][ci] [![License](https://img.shields.io/github/license/RedMadRobot/redmadrobot-android-ktx?style=flat-square)][license]
+# Gears :gear:
 
-Missing Android KTX extensions.
+[![Build Status](https://img.shields.io/github/actions/workflow/status/RedMadRobot/gears-android/main.yml?branch=main&style=flat-square)][ci]
```

Comment:
Also there is on any pr - the last ci will fall - here you will be badged with an error - although the master is green.

Maybe like this: https://github.com/RedMadRobot/gears-android/actions?query=branch%3Amain++

Reply:
No-no, the badge will be OK as it has an argument `branch=main`